EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001310	The Tree of Truth sect turned the monastery into a complete mess...
+QUEST_LV_0300_20160310_001311	We have to clean up the blood of the demons and tidy up the scripture...{nl}We have many things to do.
+QUEST_LV_0300_20160310_001312	It will be effective!{nl}Even so... If it goes wrong, you should tell me... For sure...
+QUEST_LV_0300_20160310_001313	Revelator, I am gratefully thankful to you.
+QUEST_LV_0300_20160310_001314	I've opened my new eyes because of Dominikas.{nl}Only the Divine Tree can save us...
+QUEST_LV_0300_20160310_001315	I need to save my power since we don't know when {nl}the Tree of Truth sect will rush into the monastery.{nl}I really thank you.
+QUEST_LV_0300_20160310_001316	How could we lose the monastery to the heresy like the Tree of Truth sect...{nl}I feel sorry to the goddess and I am thankful to her.
+QUEST_LV_0300_20160310_001317	I won't be deceived by the lies of the goddess anymore.{nl}You also think so, right?
+QUEST_LV_0300_20160310_001318	You really are...{nl}You might be the envoy that was sent by the goddess...{nl}
+QUEST_LV_0300_20160310_001319	What's strange is that Dominikas often visited Pagarba Chapel.{nl}It could very well mean that he may have hidden something important there.{nl}
+QUEST_LV_0300_20160310_001320	If you hide there, you will definitely manage to ambush Dominikas.{nl}I will pray for you in the name of the goddess.
+QUEST_LV_0300_20160310_001321	As we cleaned up the monastery...{nl}I have a bunch of the Soul Spheres that were left by Dominikas.{nl} 
+QUEST_LV_0300_20160310_001322	Can you remember when you broke the sinister idol?{nl}How about I put the demon souls into the sphere and purify them.{nl}
+QUEST_LV_0300_20160310_001323	I will give you the Soul Sphere if you could defeat the demons with these, please.{nl}You can put the souls into this sphere.
+QUEST_LV_0300_20160310_001324	One day, I will become the Alchemist Master!
+QUEST_LV_0300_20160310_001325	I receive help of Vaidotas until I become the Alchemist Master.{nl}It's not because I lost to Vaidotas, but I am just trying to use him!{nl}
 QUEST_LV_0300_20160310_001326	Really!!
-QUEST_LV_0300_20160310_001327	I feel something strange in the villagers. As he said, let's help the villagers.
-QUEST_LV_0300_20160310_001328	Trust the divine tree and be rescued.{nl}Our religious  body of the truthful tree is always open.
-QUEST_LV_0300_20160310_001329	The untruthful goddess, the untruthful doctrines.. Your belief on the goddess would become strong.{nl}But, that's wrongful salvation.{nl}
+QUEST_LV_0300_20160310_001327	I sense something strange in the villagers. As he said, let's help the villagers.
+QUEST_LV_0300_20160310_001328	Believe in the Divine Tree and let it save you.{nl}The door of the Tree of Truth sect is always open.
+QUEST_LV_0300_20160310_001329	Fake goddesses, untruthful doctrines... Your belief in the goddesses is strong?{nl}But that is the wrong path to salvation.{nl}
```

Comment:
"Fake goddesses and their untruthful doctrines... Your belief in them has become quite strong...{nl}But that's the wrong path to salvation."